data/wanderer/wanderers start.txt

```diff
@@ -260,113 +262,67 @@ mission "Wanderers: Translation Machine"
 			`	They respond, and Sayari translates, "They say that their species has encountered many other species in the past, and always has found those encounters to be valuable. Although..." She pauses to listen as the Wanderers continue speaking. "Although, they say that in their travels, far more often than they encounter living species, they instead only find themselves sifting through the refuse of a species that has destroyed itself through war or carelessness. They hope that you are not such a species, because..." Again she pauses as the Wanderers speak, then says, "Because, they hate to make new friends only to lose them again."`
 			
 			label end
-			`	Speaking through Sayari, you try to reassure the Wanderers that you have peaceful intentions. They speak with her for a while longer. She tells you, "They say that I will be allowed to stay here as an ambassador, and they will also let you land on most of their worlds. But they will not welcome any other Hai into their territory. I fear that they distrust the Hai because of the actions of the Unfettered, and being associated with me may harm your chances to win their favor. For your sake, we should find a different way for you to communicate with them."`
+			`	Speaking through Sayari, you try to reassure the Wanderers that you have peaceful intentions. They speak with her for a while longer. "They say that I will be allowed to stay here as an ambassador," She informs you. "They will also let you land on most of their worlds, but will not welcome any other Hai into their territory. I fear they distrust the Hai because of the actions of the Unfettered, and being associated with me may harm your chances to win their favor.`
```

Comment:
```suggestion
			`	Speaking through Sayari, you try to reassure the Wanderers that you have peaceful intentions. They speak with her for a while longer. "They say that I will be allowed to stay here as an ambassador," she informs you. "They will also let you land on most of their worlds, but will not welcome any other Hai into their territory. I fear they distrust the Hai because of the actions of the Unfettered, and being associated with me may harm your chances to win their favor.`
```